README.md

```diff
@@ -97,6 +97,8 @@ Go to `Preferences > Editor > Code Style > Java > Tabs and Indents` and update:
 
 ![Intellij imports](/intellij/intellij-indents.png)
 
+Go to `Preferences > Editor > General` and check the option `Ensure line feed at file end on Save` (under the `Other` section).
```

Comment:
I think backticks in Markdown are intended for code. I prefer to use `**strong emphasis**` when referencing UI elements (window titles, button labels, etc)

Reply:
I agree that using *emphasis* is more suitable for this example.
However, I used backticks for the sake of consistency..